packages/node_modules/@node-red/editor-client/src/js/ui/deploy.js

```diff
@@ -653,9 +657,13 @@ RED.deploy = (function() {
                     ensureUnlocked(ws.z)
```

Comment:
```suggestion
                    ensureUnlocked(ws.id)
```

We should ensure the flow is unlocked - so pass in its own id. If the flow contains any nodes, it will already have been temporarily unlocked. This handles the edge case the flow is empty.

Reply:
I had thought about this case but if it does not contain a node, there is nothing to modify right?

Reply:
we are modifying the properties of the flow itself, so yes, we are modifying it.

Reply:
I don't see a Proxy Handler on the Workspace. I put in case the `ensureUnlocked` function but I don't think if it's really used.